lambda_function.py

```diff
@@ -0,0 +1,88 @@
+import json
+import requests
+import os
+import urllib.parse
+
+
+def lambda_handler(event, context):
+    # Base URL of Gupshup's OTP API, including all constant params
+    baseURL = os.environ.get("OTP_API_BASE_URL")
```

Comment:
```suggestion
    base_URL = os.environ.get("OTP_API_BASE_URL")
```
In python code, let's keep all variables as `snake_case`!